docs/advanced_topics/images/standardizing_renditions.rst

```diff
@@ -0,0 +1,138 @@
+.. _standardizing_renditions:
+
+Standardizing Renditions
+========================
+
+The bigger your project gets, the more important it is to keep your rendition
+formats organized. So instead of having multiple variations of
+rendition-formats sprinkled throughout your code, you try to move them into a
+single source of truth, as to not keep repeating yourself.
+
+A practial example, using the popular ``srcset`` attribute for responsive images:
+
+ .. code-block:: html+Django
+
+    {% image box.image width-600 as regular %}
+    {% image box.image width-1200 as hidpi_retina %}
+    <img src="{{ regular.url }}" srcset="{{ regular.url }} 1x, {{ hidpi_retina.url }} 2x" />
+
+Repeating these filter-specs (``width-600``, ``width-1200``,…) throughout
+your project is a tad tedious, a possibility for errors and inconsistencies.
+It would be much more convenient to be able to do something along these
+lines:
+
+ .. code-block:: html+Django
+
+    {% image box.image filter_specs.medium as regular %}
+    {% image box.image filter_specs.large as hidpi_retina %}
+    <img src="{{ regular.url }}" srcset="{{ regular.url }} 1x, {{ hidpi_retina.url }} 2x" />
+
+Wagtail let's you do just that, because the filter spec can be a template variable.
+
+You can implement this any way you want. An example is illustrated below.
+
+
+Setup
+-----
+
+We are going to achieve three things:
+
+1. We will unify all filter specs in a common python file (e.g. ``settings.py``).
+2. A context processor will put these specs into our templates.
+3. As the filter specs now live in a single place (``settings.py``) you can
+   also import them in your python code. This is great if you are :ref:`image_renditions`.
+
+
+Common filter specs
+^^^^^^^^^^^^^^^^^^^
+
+You can put the filter specs anywhere you want. Highly recommended is a place
+that you can access freely from anywhere in your code. For most django
```

Comment:
I suggest dropping "Highly recommended is a place that you can access freely from anywhere in your code."